net-misc/onedrivegui-bin/onedrivegui-bin-1.0.3-r3.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+M_PN=OneDriveGUI
+
+DESCRIPTION="A simple GUI for OneDrive Linux client, with multi-account support."
+HOMEPAGE="https://github.com/bpozdena/OneDriveGUI"
+
+inherit desktop xdg-utils
+SRC_URI="
+	https://github.com/bpozdena/${M_PN}/releases/download/v${PV}/${M_PN}-${PV}_fix150-x86_64.AppImage -> ${P}.AppImage
```

Comment:
```
% pkgcheck scan --net net-misc/onedrivegui-bin
net-misc/onedrivegui-bin
  DeadUrl: version 1.1.1-r2: SRC_URI: 404 Client Error: Not Found for url: https://github.com/bpozdena/OneDriveGUI/releases/download/v1.1.1/OneDriveGUI-1.1.1-x86_64.AppImage
  RedundantVersion: version 1.0.3-r3: slot(0) keywords are overshadowed by version: 1.1.1-r2
```

If you try to go to that URL you will find it 404. Upstream seems to have renamed it to https://github.com/bpozdena/OneDriveGUI/releases/tag/v1.1.1a (note the `a` at the end).

You can run `pkgdev manifest -f` until you are able to download all the files.